did-jwk/README.md

```diff
@@ -0,0 +1,10 @@
+# did-key
+
+Rust implementation of the [did:jwk][] DID Method, based on the [ssi][] library.
```

Comment:
nitpick, these links are empty?

Reply:
I think that's a markdown thing, they're referenced at the end of the file like a bibliography